library.json

```diff
@@ -16,6 +16,9 @@
         "url": "https://github.com/Mnark/esp32-camera.git",
         "branch": "master"
     },
+    "export": {
+        "exclude": "library.json"

```

Comment:
What is it? Why did you add it? Please remove the `export` section.

Reply:
It is an option to instruct the PlatformIO Library Dependency check from attempting to install itself, which is what appears to be happening and causing the 'Access denied' issue.

Unfortunately to test it out, I need to deploy it to my repo, which is notifying you.

Reply:
Ah, I see. "Access denied" is OS related issue, not PlatformIO. You may be opened this file in text editor and OS can't remove it.